lib/grizzly/command_class/node_provisioning/delete.ex

```diff
@@ -0,0 +1,77 @@
+defmodule Grizzly.CommandClass.NodeProvisioning.Delete do
+  @moduledoc """
+  Command Options:
+
+    * `:dsks` - A list of DSK strings for provisioned devices, see `Grizzly.DSK`
+               (can be empty - the default - to delete all provisioned nodes)
+      for more details
+    * `:seq_number` - The sequence number of the Z/IP Packet
+    * `:retries` - The number of times to try to send the command (default 2)
+  """
+  @behaviour Grizzly.Command
+
+  alias Grizzly.{Packet, DSK}
+  alias Grizzly.Command.{EncodeError, Encoding}
+
+  @type t :: %__MODULE__{
+          dsks: [DSK.dsk_string()],
```

Comment:
I could be wrong here, but I think we can only either delete one DSK at a time or all DSKs.

And the `dsk_length` field is either `0` or `16`.

If no DSK is provided we set the length to `0` and if the DSK is provided we set it `16`

I am looking at `SDS13784 Z-Wave Network-Procotol Communication Class Specification.pdf` Secion `4.6.5`.

I could missing another peice of documentation though, let me know if so.

Reply:
Ok. That was embrassing. A big mis-reading on my part. Fixed

Reply:
No worries, I have done the same thing! Thanks for checking that out, I will test again here in a little bit.

Reply:
Works!